reference/api/keys.md

```diff
@@ -287,12 +284,19 @@ Date and time when the key will expire, represented in RFC 3339 format. `null` i
 
 <RouteHighlighter method="PATCH" route="/keys/{key_or_uid}"/>
```

Comment:
We are missing a body section + table for this endpoint! It should include `name` and `description`.

Reply:
I added the links for `name` and `description` in line 287, but a table would be better